tests/test_util.py

```diff
@@ -32,5 +34,8 @@ def test_load_config_valid():
 
 def test_connect_from_config_valid(valid_config):
     """Test that a valid configuration connects to appropriate connections."""
-    # TODO: Update this connection method. See #50 for more details.
-    util.connect_from_config(valid_config)
+    connections = valid_config["connections"]
+    for alias in connections.keys():
+        connect(
+            host=connections[alias]["uri"], mongo_client_class=MongoClient, alias=alias
+        )
```

Comment:
Would it be better to add a `mongo_client_class` argument to `util.connect_from_config()` with the appropriate default value so that it continues to function as expected if `mongo_client_class` is unspecified?  That way you could call `util.connect_from_config()` here and (1) slightly improve the test coverage and (2) DRY the code out a little by avoiding recreating the body of `util.connect_from_config()` in this test function.

Reply:
Great suggestion! I applied it in [9973720](https://github.com/cisagov/admiral/pull/47/commits/99737202bfa252c9077526b38daeb54fb3e07ac2) and [193ac57](https://github.com/cisagov/admiral/pull/47/commits/193ac57bcaff55fae669a045c04bbf47b30b5590).